test/fixtures/yarn-2-with-prune-plugin/.pnp.js

```diff
@@ -1,10724 +0,0 @@
-#!/usr/bin/env node
```

Comment:
Since we're deleting this file, and I assume it's auto-generated, should this also be added to `.gitignore`?

Reply:
i don't think so.  this belonged to the test fixture that covered pruning via the plugin and contained whatever files that yarn needed in place.  i deleted it though since that scenario overlapped with the [`yarn-2 fixture`](https://github.com/heroku/heroku-buildpack-nodejs/tree/83aa0deb88866cd91025f666d85b5c92a9394b9f/test/fixtures/yarn-2) once the old pruning path was  removed.